main.go

```diff
@@ -152,6 +154,8 @@ func main() {
 		}
 		basePath := filepath.Join(homeDir, "AppData", "Roaming", "gnupg")
 		handleGPG(filepath.Join(basePath, *gpg))
+	} else if *ssh != "" {
+		handlePipedSSH()
 	} else {
 		handleSSH()
 	}
```

Comment:
Use early return instead of complicated if else structure

Reply:
Done